target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -20,6 +20,7 @@ ath79_setup_interfaces()
 	tplink,tl-mr3040-v2|\
 	tplink,tl-wr703n|\
 	ubnt,bullet-m|\
+	ubnt,nanostation-ac-loco|\
```

Comment:
Nanostation AC Loco is a two-port device. Are you sure you want to only configure LAN port here?

Reply:
My bad, I confused it with full Nanostation. Please disregard this comment.